### ISSUE TITLE
Inserting text to BLOB SUB_TYPE 1 doesn't work

### ISSUE THREAD

**Issue by unlimittt** [edgardmessias/yii2-firebird#2]
Doesn't work:

``` php
$model = new Message();
$model->setAttributes(['text' => 'some text']);
$model->save();
```

Doesn't work:

``` php
$command = Yii::$app->db->createCommand('INSERT INTO MESSAGE (TEXT) VALUES(:TEXT)', [':TEXT' => 'hello',]);
$command->queryAll();
```

Does work with plain pdo:

``` php
$conn = new PDO($dsl, $dbuser, $dbpass);
$text = 'some text';
$sql = "INSERT INTO message (text) values(?)";
$q = $conn->prepare($sql);
$q->bindParam(1, $text);
$q->execute();
```

Doesn't work means it executes ok, goes to next line and suddenly executes it again and again(2-5 times) and responding nothing at the end. But in database appear 2 or 5 same records.

This is true for another yii2 firebird extension: https://github.com/srusakov/yii2-firebirddb
Maybe this is the problem with yii itself but i can't find out where is the catch.

**Comment by edgardmessias** [comment]
I'm investigating

See this: https://bugs.php.net/bug.php?id=61183

**Comment by mariuz** [comment]
I started to work on the patch , worked ok on php 5.x but i had an issue with merging with php7
So i need to rework it for 7

**Comment by edgardmessias** [comment]
Good work @mariuz

**Comment by mariuz** [comment]
@edgardmessias 

please test with php 7.1.x snapshot 
fixed by 

https://github.com/php/php-src/pull/2183/

**Comment by edgardmessias** [comment]
I need to update the tests before

**Comment by edgardmessias** [comment]
Fixed in 5886177
